gemfiles/redis-activesupport.gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem "redis-activesupport"
+
+gemspec path: "#{__dir__}/.."
```

Comment:
Good discussion of this issue - http://yehudakatz.com/2010/04/17/ruby-require-order-problems/ . Wondering if there is a better solution now...